create_infotexts.py

```diff
@@ -495,14 +492,19 @@ def process_depicted_place(self, place_string, places_mapping, desc_string):
         self.data["depicted_place"] = place_as_wikitext
         
     
-    def process_keywords(self, item, mapping):
+    def process_keywords(self, mapping):
         """
         Add potential content categories from column <Nyckelord>.
         
         Populates self.content_cats
         
         :return: None
```

Comment:
No need to mention the `:return: None` (since you mention that it populates self.content_cats)